provider/core/src/marker.rs

```diff
@@ -563,9 +564,20 @@ impl DataMarkerInfo {
     pub const fn from_path(path: DataMarkerPath) -> Self {
         Self {
             path,
+            fallback_config: LocaleFallbackConfig::default(),
             is_singleton: false,
+            #[cfg(feature = "export")]
             attributes_domain: "",
-            fallback_config: LocaleFallbackConfig::default(),
+        }
+    }
+
+    /// TODO
+    #[cfg_attr(not(feature = "export"), allow(unused_variables))]
```

Comment:
Why `unused_variables`?

Reply:
because the parameter is unused without the feature